content/docs/atom-guide/features/_index.md

```diff
@@ -96,20 +96,20 @@ Atom has a unified mesh format with support for several mesh types:
 *Related to: [Mesh component](/docs/user-guide/components/reference/atom/mesh), [Mesh Feature Processor API](/docs/api/gems/Atom/class_a_z_1_1_render_1_1_mesh_feature_processor.html), [Skinned Mesh Feature Processer API](/docs/api/gems/Atom/class_a_z_1_1_render_1_1_skinned_mesh_feature_processor_interface.html)*  
 
 
-## Processing and Post FX
+## Post-processing effects
 
 Tone Mapping and Color Grading   
 : Techniques that adjust the overall lighting and color in a scene. Tone mapping simulates high dynamic range (HDR) by mapping one set of colors to another set that has a higher range. Color grading adjusts the color output of the render. There are two color grading modes: HDR Color Grading and LDR Color Grading. Tone mapping and color grading can be configured in O3DE using the **Display Mapper**. You can also view a demo of tone mapping and color grading in the Atom Sample Viewer. 
```

Comment:
Tonemapping and color grading are only related because of them happening closely together in the render pipeline (Display Mapper), they do both later colors but with different purpose.

I think we should start this section describing what the 'Display Mapper' is:

The Display Mapper configures how the renderer performs toenampping for viewing the production on screen.  

Tone Mapping is the process of converting the tonal values of an image from a high range (HDR) to a lower one.  Atom renders in a 16-bit HDR framebuffer, applying tonemapping makes the rendered image suitable to be viewed on a wide variety of digital screen technologies; from standard 48-nit LDR displays (sRGB/rec.709) to various HDR display types that may be 1000-nit or brighter.

The default tonemapping in Atom is called ACES (https://acescentral.com/). ACES standardizes and simplifies color management by maintaining the artistic integrity, fidelity and best reproduction of color from renderer to any display, as well as across the toolchain as many modern DCC apps can be configured to also use ACES managed color and tonemapping.  This provides a better and more consistent experience of working with color in many roles, whether texture and material artist, lighting specialists, cinematic directors, etc.

Reply:
Color Grading
Color grading is the process of altering the visual appearance of an image for presentation in different environments on different devices, or for artistic and stylistic enhancement. Various attributes of an image such as contrast, color, saturation, detail, black level, and white point may be enhanced.  Color Grading and Tone Mapping often go hand in hand as they both contribute to the color and tone of the image displayed on the users device.  In O3DE users cane perform HDR color grading within the internals of the render pipeline via the **PostFX** system and **Look Modification** component (using a color look up table, LUT), or a single global LDR color grading on the final display output can be applied with the ""Display Mapper** level component.